src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -1212,18 +1223,19 @@ class DeviceArgument(cli.Argument):
     )
     DEVICE_NAME = cli.ArgumentProperties(
         key="device_name",
-        flags=("--name",),
+        flags=("--name", "-n"),
         description="Name of the Device",
         argparse_kwargs={"required": True},
     )
     DEVICE_NAME_OPTIONAL = cli.ArgumentProperties.duplicate(
         DEVICE_NAME,
         argparse_kwargs={"required": False},
     )
-    DEVICE_UDID = cli.ArgumentProperties(
-        key="device_udid",
-        flags=("--udid",),
-        description="Device ID (UDID)",
+    DEVICE_UDIDS = cli.ArgumentProperties(
+        key="device_udids",
+        flags=("--udid", "-u"),
+        type=Types.DeviceUdidsArgument,
+        description=f"Device ID (UDID), for example: {Types.DeviceUdidsArgument.example_value}",
         argparse_kwargs={"required": True},
```

Comment:
It cannot be required argument if we want to allow environment variable overriding.